sashimi-webapp/src/logic/filemanager/history.js

```diff
@@ -0,0 +1,52 @@
+const previousHistory = [];
```

Comment:
I feel that can have a brief description on what this history file actually does (which is to "navigate through recently visited directories" - this one i got from Microsoft about their file explorer one :D). Would make it clearer and less ambiguous! :) Also maybe instead of "...History" can use "...Directory"/"...Path"/"...Location" these names might be more relevant for the functionality of this component! :D (e.g. "previousDirectory", "forwardDirectory", "currDirectory" etc.)